test/parallel/test-console-clear.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+
+const stdoutWrite = process.stdout.write;
+
+// The sequence for moving the cursor to 0,0 and clearing screen down
+const check = '\u001b[1;1H\u001b[0J';
+
+let buf = '';
+
+// Fake TTY
+process.stdout.isTTY = true;
```

Comment:
This should probably (artificially) test the case where `isTTY = false` as well.